<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAddRoom</class>
 <widget class="QDialog" name="dlgAddRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>224</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>224</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dodaj pokój</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbSize">
     <property name="title">
      <string>Rozmiar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbSizeWidth">
        <property name="text">
         <string>Szerokość:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbSizeWidth">
        <property name="suffix">
         <string> kratek</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbSizeHeight">
        <property name="suffix">
         <string> kratek</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbSizeHeight">
        <property name="text">
         <string>Wysokość:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="spVertical">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbFill">
     <property name="title">
      <string>Wypełnienie</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbFill">
        <property name="text">
         <string>Wypełnij pokój meblami</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbFillThings">
        <property name="text">
         <string>Pomieść rzeczy:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sbFillThings">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbFillPeople">
        <property name="text">
         <string>Pomieść osób:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbFillPeople">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="spVertical2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="layHorizontal">
     <item>
      <spacer name="spHorizontal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbSizeWidth</tabstop>
  <tabstop>sbSizeHeight</tabstop>
  <tabstop>cbFill</tabstop>
  <tabstop>sbFillPeople</tabstop>
  <tabstop>sbFillThings</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgAddRoom</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFill</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbFillPeople</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFill</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbFillThings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
